iac-extensions/arm/src/test/resources/checks/HardCodedCredentialsCheck/HardCodedCredentialsCheck.bicep

```diff
@@ -0,0 +1,24 @@
+resource sqlServer 'Microsoft.Sql/servers@2023-05-01-preview' = {
+  name: 'noncompliant'
+  properties: {
+    administratorLogin: 'admin' // Noncompliant {{Revoke and change this secret, as it might be compromised.}}
+//  ^^^^^^^^^^^^^^^^^^^^^^^^^^^
+    administratorLoginPassword: 'password' // Noncompliant
+    password: 'password' // Noncompliant
+    secret: 'password' // Noncompliant
+    adminPassword: 'password' // Noncompliant
+    adminUsername: 'admin' // Noncompliant
```

Comment:
Please also add some non-sensitive keys to be sure we are not raising on each key inside `properties`